src/views/home/home.view.tsx

```diff
@@ -26,40 +22,31 @@ export const Home = (): JSX.Element => {
   const env = useEnvContext();
   const { formData, setFormData } = useFormContext();
   const { connectedProvider } = useProvidersContext();
-  const [depositLimitModal, setDepositLimitModal] = useState<ModalState<DepositLimitModalData>>({
+  const [depositWarningModal, setDepositWarningModal] = useState<ModalState<FormData>>({
     status: "closed",
   });
 
-  const onSubmitForm = (formData: FormData) => {
+  const onSubmitForm = (formData: FormData, hideDepositWarning?: boolean) => {
+    if (hideDepositWarning) {
+      setIsDepositWarningHidden(hideDepositWarning);
+    }
     setFormData(formData);
     navigate(routes.bridgeConfirmation.path);
   };
 
-  const onCheckDepositLimitAndSubmitForm = (formData: FormData) => {
-    if (env && env.areDepositLimitsEnabled && formData.from.key === "ethereum") {
-      const { from, token } = formData;
-      const depositLimits = DEPOSIT_LIMITS[from.chainId][token.address];
-
-      if (depositLimits === undefined) {
-        setDepositLimitModal({
-          data: { formData, type: "unknown-limit" },
-          status: "open",
-        });
-      } else {
-        const softLimit = ethersUtils.parseUnits(depositLimits.soft.toString(), token.decimals);
-        const hardLimit = ethersUtils.parseUnits(depositLimits.hard.toString(), token.decimals);
+  const onCheckShowDepositWarningAndSubmitForm = (formData: FormData) => {
+    const isDepositWarningHidden = getIsDepositWarningHidden();
```

Comment:
This PR introduces several references to "[is/getIs/setIs]DepositWarningHidden". This wording is misleading since this does not mean it's "hidden" but "dismissed by the user".

I'd replace all references to `[is/getIs/setIs]DepositWarningHidden` by `[is/getIs/setIs]DepositWarningDismissed`

<img width="779" alt="image" src="https://user-images.githubusercontent.com/15896177/227911223-510191a6-86ff-4a2e-98dc-b1f63b64f4bb.png">

Reply:
Right, done